open-metadata-implementation/user-interfaces/presentation-server/cra-client/.env.sample

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0 
+# Copyright Contributors to the ODPi Egeria project. 
+
+# This file is a sample .env file that can be used by developers.
+# Copy this file to a file called .env and modify the values as required
+HTTPS=true
```

Comment:
be good to add comments on what the files are for (I realise, but for benefit of others)

Reply:
add clarifying comments